src/index.ts

```diff
@@ -2,12 +2,81 @@
 
 import chalk from 'chalk'
 import figlet from 'figlet'
-import clear from 'clear'
 import { Command } from 'commander'
 
+import { loadProcesses } from './lib/process/index.js'
+
 const program = new Command()
+const example: string = JSON.stringify([{
+  name: 'A test',
+  version: 1,
+  program: [  
+    { restriction: { SenderOwnsAllInputs: {} }} ,
+    { restriction: { SenderHasInputRole: 
+      {
+        index: 0,
+        roleKey: 'Supplier',
+      },
+    }},
+    { op: 'and'},
+    { restriction: { FixedOutputMetadataValueType: {
+        index: 0,
+        metadataKey: 'SomeMetadataKey',
+        metadataValueType: 'Literal',
+      }
+    }},
+    { restriction:{ FixedOutputMedataValueType:
+      {
+        index: 0,
+        metadataKey: 'SomeOtherMetadataKey',
+        metadataValueType: 'File',
+      },
+    }},
+    { op: 'and'},
+    { op: 'and'},
+  ],
+}])
+ 
+program
+  .name('process management')
+  .description('a command line interface for managing chain processes')
+  .version('1.4.0')
+
+
+program.command('create')
+  .description('A command for persisting process flows onto the chain')
+  .option('-d, --dryRun <bool>', 'performs a dry run', false)
+  .option('-h, --host <string>', 'substrate blockchain host address or FQDM, default - "localhost"', 'localhost')
+  .option('-p, --port <number>', 'specify host port number if it is not a default, default - 9944', '9944')
+  .option('-u, --user <string>', 'spefify substrate blockhain user URI, default - "//Alice"', '//Alice')
+  .argument('<string>', `takes JSON as string example: '${example}'`)
+  .action(async (data: string, options: { dryRun: boolean, port: number, user: string, host: string }) => {
+    // README it looks like it goes with thedefault values, not sure comming and leaving a comment
+    // TODO figure out why it's not parsing options
```

Comment:
this is fixed now based off what you said in standup?

Reply:
yes!, comment has been removed